humanguide/other-aspects/06-scaling-images.rst

```diff
@@ -0,0 +1,26 @@
+Scaling images
+==============
+
+One important aspect for app convergence is the images and how to scale
+them. From app icon to in-app images can be done in
+`svg <https://en.wikipedia.org/wiki/Scalable_Vector_Graphics>`__
+(scalable format) using software like
+`Inkscape <https://inkscape.org/>`__.
```

Comment:
"Scaling images is important in a convergent app. App icons and images can be created in scalable formats to improve their ability to scale."